ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -522,6 +530,14 @@ VAV_HWZTC_CHWZTC_DTM:
     - CHWZTC
     - DTM
 
+VAV_MSVO_PDDCV:
```

Comment:
Can you confirm that there is no sort of temperature or CO2 monitoring on the VAV?

Reply:
Hello, I confirm that the regulation is only based on the motion in the room.